src/main/kotlin/com/sourcegraph/cody/edit/widget/LabelHighlight.kt

```diff
@@ -15,11 +17,23 @@ class LabelHighlight(private val color: Color) {
    * @param textHeight the height of the text.
    */
   fun drawHighlight(g: Graphics2D, x: Float, y: Float, textWidth: Int, textHeight: Int) {
+    // Draw shadow
```

Comment:
Make sure you try this with 6-point and 36-point Editor fonts.

Reply:
Good point. Its not pixel perfect at 6 nor 36, but still maintains a decent and functional look considering that those are extremes sizes

<img width="1201" alt="Screenshot 2024-05-22 at 18 22 09" src="https://github.com/sourcegraph/jetbrains/assets/7814431/151eee80-c070-48cc-af5e-0199525e3f98">
<img width="399" alt="Screenshot 2024-05-22 at 18 22 24" src="https://github.com/sourcegraph/jetbrains/assets/7814431/43dfc26c-25fc-4ab3-813f-871949f6e83f">